modules/node-observability-install-web-console.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/understanding-node-observability-operator.adoc
+
+:_content-type: PROCEDURE
+[id="install-node-observability-using-web-console_{context}"]
+= Installing the Node Observability Operator using the web console
+
+You can install the Node Observability Operator from the {product-title} web console.
+
+.Prerequisites
+
+* You have access to the cluster with `cluster-admin` privileges.
+* You have access to the {product-title} web console.
+
+.Procedure
+
+. Log in to the {product-title} web console.
+. In the Administrator's navigation panel, expand *Operators* → *OperatorHub*.
```

Comment:
"expand" could be confusing here. You might want to say "Navigate to", or "Click".

Reply:
@stevsmit IBM style guide for **Navigation tree** element says:

> Verb: Generally, use click to refer to selecting items from the navigation tree (for exceptions,
see Verbs to use). **Use expand to refer to clicking a twistie or a plus sign (+)** to display the content. Use collapse to refer to clicking a twistie or a minus sign (-) to hide the content.

wdyt ?

Reply:
Thanks for sharing!